examples/poisson.pct.py

```diff
@@ -0,0 +1,236 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     custom_cell_magics: kql
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.2
+#   kernelspec:
+#     display_name: base
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Count data regression with the Poisson likelihood
```

Comment:
Could we name this something short like "Count data regression"? It renders nicer in the docs' table of contents this way.